roofit/roofitcore/inc/RooNLLVarNew.h

```diff
@@ -0,0 +1,46 @@
+#ifndef RooNLLVarNew_h
+#define RooNLLVarNew_h
+
+#include "RooAbsPdf.h"
+#include "RooAbsReal.h"
+#include "RooTemplateProxy.h"
+
+#include "RooBatchCompute.h"
+
+class RooNLLVarNew : public RooAbsReal {
+
+public:
+  RooNLLVarNew(){};
+  RooNLLVarNew(const char *name, const char *title, RooAbsPdf &pdf);
+  RooNLLVarNew(const RooNLLVarNew &other, const char *name = 0);
+  virtual TObject *clone(const char *newname) const override { return new RooNLLVarNew(*this, newname); }
+
+  RooArgSet *getParameters(const RooArgSet *depList, Bool_t stripDisconnected = kTRUE) const override;
+
+  virtual Double_t defaultErrorLevel() const override
+  {
+    // Return default level for MINUIT error analysis
+    return 0.5;
+  }
+
+  inline RooAbsPdf* getPdf() const { return &*_pdf; }
+  void computeBatch(double* output, size_t nEvents, rbc::DataMap& dataMap) const override;
+  double reduce(const double* input, size_t nEvents) const;
+
+protected:
+  RooTemplateProxy<RooAbsPdf> _pdf;
+
+  double getValV(const RooArgSet *normalisationSet = nullptr) const override;
+
+  double evaluate() const override;
+
+  RooSpan<double> evaluateSpan(RooBatchCompute::RunContext &evalData, const RooArgSet *normSet) const override;
```

Comment:
This does not complile , due to a change in RooAbsArg::getParameters in the master. 
It should be, :
 virtual bool getParameters(const RooArgSet* observables, RooArgSet& outputSet, bool stripDisconnected=true) const;